css/style.css

```diff
@@ -0,0 +1,1544 @@
+/* General reset and classes */
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  position: relative;
+}
+
+ul,
+li {
+  margin: 0;
+}
+
+.w-100 {
+  width: 100%;
+}
+
+.flexspacebetween {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.flexcenter {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* styling for large screen */
+
+/* top navigation styling */
+.navtop {
+  background-color: black;
+  padding: 10px;
+  position: relative;
+}
+
+.navlist {
+  display: flex;
+  list-style-type: none;
+}
+
+.navlist li {
+  margin-right: 15px;
+}
+
+.navlist a {
+  text-decoration: none;
+  color: #aaa;
+}
+
+.footbottom a {
+  text-decoration: none;
+  color: rgba(167, 169, 169, 1);
+}
+
+.footnav a {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: rgba(167, 169, 169, 1);
+  margin-right: 10px;
+}
+
+.flexed-desc a {
+  text-decoration: none;
+}
+
+.sec-4-big-pic-third a {
+  text-decoration: none;
+}
+
+.sec-4-big-pic-second a {
+  text-decoration: none;
+}
+
+.sec-4-big-pic-first a {
+  text-decoration: none;
+}
+
+.textfoot a {
+  font-family: "Varela Round", sans-serif;
+  font-size: 12px;
+  color: rgba(36, 36, 36, 0.9);
+}
+
+.title a {
+  font-family: "Russo One", sans-serif;
+  text-decoration: none;
+  color: #f42;
+  font-size: 24px;
+  font-weight: bolder;
+}
+
+.titleandlinks {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 20px;
+  align-items: baseline;
+}
+
+.titleholder {
+  margin-right: auto;
+}
+
+.title-side {
+  margin: 0 6px;
+}
+
+.p-holder {
+  display: flex;
+}
```

Comment:
- [ ] OPTIONAL ➡ `display: flex` is repated 17 times troughout your code. I recommend you guys to create a `.d-flex { display: flex }` class to handle that rule, in order to keep your code DRY.